custom_components/eo_mini/api.py

```diff
@@ -151,6 +158,8 @@ async def _async_api_wrapper(
                 text = await response.read()
                 _LOGGER.info("Response: %r", text)
                 return text
+        if response.status == 404:
```

Comment:
A 404 would be some kind of programming error, I'd have thought... The only things which could cause that are either we got our URL wrong, or EO changed the service. Either way, it's an error rather than no response, in my opinion.

Reply:
Hey, Here it's not a programming error. 404  "Not Found", is used to indicate that the requested resource is not present, in this case it indicates the status of the vehicle with calling the liveness URL `/api/session/alive`.

Reply:
Yeah this seems to be exactly the case. A 'session' is equal to a car being plugged in to the device and the response code from the `alive` path indicates this via response code.